### ISSUE TITLE
Using your own uikit themes and LESS overrides with uikit-react?

### ISSUE THREAD

**Issue by SVAR7BERG** [stipsan/uikit-react#451]
Hey, how am I supposed to create an own style or theme for uikit when using uikit-react? I'm sure it works just like when I clone the uikit project in my project folder and create a style but in the case of uikit-react I dont know where to put my own uikit style. Any help appreciated :)

**Comment by stipsan** [comment]
Hi,

uikit-react itself leaves it up to you how you load uikit css.
The way we load our own theme for our demo site http://uikit-react.io/ is by using webpack and the https://github.com/webpack-contrib/less-loader

The less-loader configuration is nothing special: https://github.com/stipsan/uikit-react/blob/master/.storybook/webpack.config.js#L24

And the uikit styling itself is loaded here: https://github.com/stipsan/uikit-react/blob/master/.storybook/uikit.less
And imported into the page here: https://github.com/stipsan/uikit-react/blob/master/.storybook/config.js#L1

In conclusion: if you follow the official documentation for how to create your own uikit theme with overrides and load it using webpack it should just work 😃

**Comment by SVAR7BERG** [comment]
Thanks for your reply! I guess I got the Idea and will try it soon.
